### ISSUE TITLE
Bug: Windows checkout in root of drive omits the required slash

### ISSUE THREAD

**Issue by davidsmoot** [gitextensions/gitextensions#4637]
Super minor but worth reporting, probably an easy fix.  

Tried to clone a repository into the root of my SD card on my laptop (d:).

Right click and select the clone option from the context menu.  URL was already on the clipboard and is populated correctly.  Click OK without changing any options.  

The clone fails with because the path is not a proper Dos path, it is "D:".  Add a slash to make it "D:\" and the clone succeeds.

Git Extensions 2.51 on Windows 10 64 bit.

**Comment by RussKie** [comment]
Can it be linked to #4406?
/cc: @loop-evgeny

**Comment by loop-evgeny** [comment]
Yep, sounds likely. Would be good to see the exact error, though.

**Comment by loop-evgeny** [comment]
I can repro this, but only from the Windows Explorer context menu. When cloning the Start, Clone Repository menu the destination is "C:\", as expected. So I don't think it's related to #4406.

**Comment by GintasS** [comment]
Abandoning this because of the age and lack of discussion.
